validate/validate.go

```diff
@@ -158,7 +158,7 @@ func (v *Validator) CheckPlatform() (msgs []string) {
 		"darwin":    {"386", "amd64", "arm", "arm64"},
 		"dragonfly": {"amd64"},
 		"freebsd":   {"386", "amd64", "arm"},
-		"linux":     {"386", "amd64", "arm", "arm64", "ppc64", "ppc64le", "mips64", "mips64le", "s390x"},
+		"linux":     {"386", "amd64", "arm", "arm64", "mips", "mipsle", "mips64", "mips64le", "ppc64", "ppc64le", "s390x"},
```

Comment:
> For the things you said I don't have any idea, do you have any good idea?

For the alphabetization, [the order described here][1] is:

    … "arm64", "mips", "mips64", "mips64le", "mipsle", "ppc64", …

For the inconsistency between `--arch` and `--seccomp-arch`, I wouldn't do anything unless the maintainers feel like something needs to be done.  I was just bringing the issue to their attention in case they wanted to act.

[1]: https://github.com/opencontainers/runtime-tools/pull/368#discussion_r112497741